### PR TITLE
Updating help to incorporate changes to cmdlets.

### PR DESCRIPTION
**Pull request by ravbhatnagar**
Removing Tag
Removing OutputObjectFormat
Updating examples

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ravbhatnagar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@ravbhatnagar need some signoff from the resource manager owners

@vivsriaus Can you please sign off on these changes

File: src/ResourceManager/Resources/Commands.Resources/Microsoft.Azure.Commands.Resources.dll-Help.xml [hunk 1]
```diff
@@ -4434,18 +4434,6 @@ NewApplication                                                61b5d8ea-fdc6-40a2
 			</dev:type>
 			<dev:defaultValue></dev:defaultValue>
 		</command:parameter>
-		<command:parameter required="false" variableLength="false" globbing="false" pipelineInput="true (ByPropertyName)" position="named">
```
Comment (vivsriaus) [inline src/ResourceManager/Resources/Commands.Resources/Microsoft.Azure.Commands.Resources.dll-Help.xml]: @ravbhatnagar Why are we removing the DeploymentName?

File: src/ResourceManager/Resources/Commands.Resources/Microsoft.Azure.Commands.Resources.dll-Help.xml [hunk 2]
```diff
@@ -4494,78 +4470,6 @@ NewApplication                                                61b5d8ea-fdc6-40a2
 			</dev:type>
 			<dev:defaultValue></dev:defaultValue>
 		</command:parameter>
-		<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="true (ByPropertyName)" position="named">
```
Comment (vivsriaus) [inline src/ResourceManager/Resources/Commands.Resources/Microsoft.Azure.Commands.Resources.dll-Help.xml]: TemplateParameter, TemplateFile and TemplateUri still exists, right @ravbhatnagar ?

**Comment by vivsriaus** [comment]
@markcowl LGTM

**Pull request merged**